src/app/components/top/reglas/reglas.component.ts

```diff
@@ -147,21 +147,34 @@ export class ReglasComponent {
     }
 
     verificarRegla() {
-        if (this.reglas?.length) {
-            this.reglaCorrecta = this.reglas.every(regla => regla.destino && regla.destino.organizacion && regla.destino.prestacion && regla.origen.organizacion && regla.origen.prestaciones?.length);
+        this.reglaCorrecta = this.reglas?.every(regla => regla.destino?.organizacion && regla.destino?.prestacion);
+    }
+
+    preSave() {
+        const reglasSinPrestacion = this.reglas?.some(regla => regla.origen.organizacion && !regla.origen.prestaciones?.length);
+        if (reglasSinPrestacion) {
+            this.plex.confirm('Existen reglas sin prestación origen. Si continúa serán eliminadas.', '¿Desea continuar?').then(respuesta => {
+                if (respuesta) {
+                    this.reglas = this.reglas?.filter(regla => regla.origen.organizacion && regla.origen.prestaciones?.length);
+                    this.verificarRegla();
+                    this.save();
+                } else {
+                    return;
+                }
+            });
         } else {
-            this.reglaCorrecta = false;
+            this.save();
         }
     }
 
-    onSave($event) {
-        if (this.reglas?.length) {
+    save() {
+        if (this.reglas.length) {
             if (this.reglaCorrecta) {
```

Comment:
Estaría bueno acomodar el código,  verificarRegla debería devolver un booleano en vez de asignar una variable para su posterior uso. Ver todos los lugares donde se usa. 

```suggestion
            if (this.verificarRegla()) {
```